bay-services/firehose-fetcher.yaml

```diff
@@ -0,0 +1,6 @@
+services:
+- hash: 86e20fcadb644b5a04595be63f4638bd4cb6f95e 
```

Comment:
Do we have a build with this hash in the registry? I don't see any builds in CentOS CI.

Reply:
This hash would do the trick: `660c81f4c88dcc85be5e8795f944a366ccb346d0`

Reply:
Thanks for pointing out! Problem was that scripts which I had in repo were not functional so that's why no image was built.